package-lock.json

```diff
@@ -7843,6 +7851,14 @@
                 "node": ">=6.9.0"
             }
         },
+<<<<<<< HEAD
+=======
+        "node_modules/geolib": {
+            "version": "3.3.1",
+            "resolved": "https://registry.npmjs.org/geolib/-/geolib-3.3.1.tgz",
+            "integrity": "sha512-sfahBXFcgELdpumDZV5b3KWiINkZxC5myAkLk067UUcTmTXaiE9SWmxMEHztn/Eus4JX6kesHxaIuZlniYgUtg=="
+        },
+>>>>>>> develop-m
```

Comment:
Looks like this conflict wasn't resolved.  There are 2 more of these as well.

Reply:
Geeeeze. I don't understand how that could have happened, but `grep -lri '<<<<<<<'` says package-lock.json was the only in-tree file with that problem. Fixed and force-pushed.